i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/system-operation/offline-installation.md

```diff
@@ -0,0 +1,3 @@
+# KubeVela 离线部署手册
+
+Currently, it's only available in Chinese.
```

Comment:
Please also finish the en version. BTW, this is the place of Chinese doc.

Reply:
Fixed, thanks.